server/src/main/java/edu/jhu/server/Game.java

```diff
@@ -477,6 +478,20 @@ private void handleJoinRequest(JSONObject request, Player author) {
   	if (this.players.size() == 3 && timer == null) {
   		timer = new Timer();
   		
+      // send notification each minute that the game will start in x minutes
+      for (int i = 60 * 1000; i < Constants.START_GAME_AFTER_MS; i += 60 * 1000) {
```

Comment:
You could also just do a period timer task, but then would have to have a stop condition. Probably would be a cleaner solution, but I think this method is acceptable as well.